README.md

```diff
@@ -92,17 +92,24 @@ The milestone needs testing on all platforms. Please report issues when found.
     - D3 Open Source compiles level scripts in their own hogfiles. Make sure you copy and overwrite `d3-{platform}.hog`.
 
 ## Building
-#### Building - Windows
-1. Make sure that you have Git and Visual Studio 2022 with the “Desktop development with C++” workload and the “C++ MFC for latest v143 build tools (x86 & x64)” component. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
+### Dependencies
+The build process uses [**CMake**](https://cmake.org/) and, by default, [**Ninja**](https://ninja-build.org/). You must install these; the project cannot locate them for you. The source code depends on [**SDL2**](https://github.com/libsdl-org/SDL) and [**zlib**](https://github.com/madler/zlib). You can supply these dependencies yourself via your system's library management tools, or the build system can locate the dependencies for you using [vcpkg](https://github.com/microsoft/vcpkg), a cross-platform dependency-management system developed by Microsoft. The official builds source their dependencies from vcpkg.
 
+### Installing and using vcpkg
+* When building for Windows, vcpkg is already installed and configured when using the Developer Tools Command Prompt.
+* For Android, Linux or Mac (or non-standard-Windows) configurations, you will need to install vcpkg locally by cloning https://github.com/microsoft/vcpkg and setting your `VCPKG_ROOT` env var to the repository location. With this environment variable set, the build will automatically locate its dependencies.
```

Comment:
Android is not technically supported yet

Reply:
> Is there a reason we don't set the toolchain in the presets?

This is an old comment, but to close the loop on it: Setting it unconditionally would make all builds unconditionally use vcpkg, which was discovered to be an anti-goal.

> Android is not technically supported yet

_yet_

Reply:
I don’t think that we should mention Android here until we actually support Android. It would make it look like the `main` branch supports Android even though it doesn’t support Android. Also, there’s no guarantee that the `main` branch will support Android. [Your other PR](https://github.com/DescentDevelopers/Descent3/pull/459) might get accepted or it might get rejected. We might even decide that the upstream version of Descent 3 should never support Android and that Android support should be done in a fork. I don’t think that any of those scenarios are likely, but they are possible.